modules/simple_forms_api/app/services/simple_forms_api/notification/email.rb

```diff
@@ -87,7 +30,7 @@ def initialize(config, notification_type: :confirmation, user: nil, user_account
       def send(at: nil)
         return unless flipper?
 
-        template_id = TEMPLATE_IDS[form_number][notification_type]
+        template_id = TEMPLATE_IDS[form_number][notification_type.to_s]
```

Comment:
We have to convert to string here because the YAML file encodes this as a symbol while our old hash had strings for these keys.